src/vkcapture.c

```diff
@@ -928,4 +928,15 @@ void obs_module_unload()
 }
 
 OBS_DECLARE_MODULE()
+OBS_MODULE_AUTHOR("David Rosca <nowrep@gmail.com>")
 OBS_MODULE_USE_DEFAULT_LOCALE(PLUGIN_NAME, "en-US")
+
+MODULE_EXPORT const char *obs_module_name(void)
+{
+    return obs_module_text("GameCapture");
```

Comment:
Not sure if this should be translated.

Reply:
Can do that as well. I was simply adapting to what I saw being done in other plugins. Most don't define this function at all, and the one who do seem to return a translated text.

Reply:
It's only `obs-websocket` in obs-studio repo and that one returns untranslated text.

Reply:
Searching GitHub, it seems quite many do a translated text: https://github.com/search?q=obs_module_name&type=code

But I don't really mind whether it's translated. There isn't any guideline from OBS side either :shrug:. Will change this to use the `PLUGIN_NAME` constant then.

Reply:
I'm not sure where this string gets displayed, but in any case "Game Capture" isn't name of this plugin.